compiler/src/org.graalvm.compiler.java/src/org/graalvm/compiler/java/LambdaUtils.java

```diff
@@ -51,6 +51,8 @@
 public final class LambdaUtils {
     private static final Pattern LAMBDA_PATTERN = Pattern.compile("\\$\\$Lambda\\$\\d+[/\\.][^/]+;");
     private static final char[] HEX = "0123456789abcdef".toCharArray();
+    public static final String SPLIT_BY_LAMBDA = "\\$\\$Lambda\\$";
```

Comment:
This can be expressed as `Pattern.quote(LAMBDA_FUNCTIONS_NAME_PATTERN)`.

Reply:
LAMBDA_SPLIT_PATTERN would be better.

Reply:
> This can be expressed as `Pattern.quote(LAMBDA_FUNCTIONS_NAME_PATTERN)`.

It cannot. LambdaUtils class is needed in runtime, so this should stay String in order to finish build process successfully.

Reply:
> LAMBDA_SPLIT_PATTERN would be better.

Changed as suggested.